packages/fxa-graphql-api/src/config.ts

```diff
@@ -41,6 +41,12 @@ const conf = convict({
       default: 'http://localhost:9000/v1',
     },
   },
+  contentfulApiKey: {
+    doc: 'GraphQL Content API key for Contentful hCMS to fetch RP-provided content',
+    format: String,
+    env: 'CONTENTFUL_API_KEY',
```

Comment:
In `fxa-auth-server`, other API keys are of the form `*_APIKEY`, but I prefer to split on the word. Happy to change it to be consistent with env vars elsewhere. No precedent yet for `fxa-graphql-api`.

Reply:
Not sure how accurate this wacky <kbd>grep</kbd> is, but this might be our current API_KEYs:

<details>
<summary><kbd>git grep -Pno "\b[A-Z_]+_API_?KEY" | sort | uniq</kbd></summary>

FILE | VAR
:----|:----
packages/fxa-admin-server/src/config/index.ts:250 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-admin-server/src/config/index.ts:556 | `BASKET_API_KEY`
packages/fxa-auth-server/config/index.ts:229 | `GOOGLE_MAPS_APIKEY`
packages/fxa-auth-server/config/index.ts:902 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-auth-server/scripts/clean-up-old-ci-stripe-customers.js:20 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-auth-server/scripts/clean-up-old-ci-stripe-customers.js:51 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-auth-server/scripts/find-stripe-sync-issues.js:36 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-auth-server/scripts/find-stripe-sync-issues.js:47 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/settings/external_links.js:60 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/sign_up.js:263 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/subscriptions.js:109 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/subscriptions.js:138 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/subscriptions.js:46 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/subscriptions.js:74 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/subscriptions.js:84 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/support.js:119 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/support.js:45 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-content-server/tests/functional/support.js:67 | `SUBHUB_STRIPE_APIKEY`
packages/fxa-graphql-api/src/config.ts:47 | `CONTENTFUL_API_KEY`
packages/fxa-payments-server/server/config/index.js:443 | `STRIPE_API_KEY`
</details>